configuration/display_rotation.md

```diff
@@ -0,0 +1,35 @@
+## How to rotate your display
+
+The options to rotate the display of your Raspberry Pi depend on which display driver software it is running, which may also depend on which Raspberry Pi you are using. 
+
+### Fake or Full KMS graphics driver (Default on Pi4)
+
+If you are running the Raspberry Pi desktop then rotation is achieved by using the `Screen Configuration Utility` from the desktop `Preferences` menu. This will bring up a graphical display of the display or displays attached. Right click on the display you wish to rotate and select the required option.
```

Comment:
"display of the display" looks weird :wink: 
Maybe change it to "representation of the display"? And maybe it's worth expanding "attached." to "connected to your Raspberry Pi." ?